### ISSUE TITLE
Log In process doesn't work

### ISSUE THREAD

**Issue by dealpete-work** [deltachat/deltachat-desktop#1859]
- Operating System (Linux/Mac/Windows/iOS/Android): MacOS
- Delta Chat Version: 1.12.0
- Expected behavior: see below
- Actual behavior: see below
- Steps to reproduce the problem: download .dmg version of Delta Chat, install it, run it, fill in email details, click "Log In"
- Screenshots:
![Screen Shot 2020-08-17 at 9 43 53 PM](https://user-images.githubusercontent.com/61948804/90456215-eea89100-e0d2-11ea-8cc2-ec48dc22fbd5.png)
- Logs: Not sure how to produce logs

Hi,

I downloaded the app for the first time today. When I open it, I see a screen asking for my email address and password. I put those in and press "Log In". A popup appears with a status bar and a "CANCEL" buts. The status bars fills to about half way, the empties itself. Nothing further happens if I leave it going. There's no feedback about what might have gone wrong.

I tried both my Google account and Fastmail accounts, same behaviour with both.

Regards,
Peter

**Comment by dealpete-work** [comment]
I figured out what was going wrong. Both my email addresses are on custom domains - I use xxx@peterdeal.ca instead of xxx@fastmail.com and my work email is peter@harplabs.com, instead of xxx@gmail.com

When I put in xxx@fastmail.com a helpful message popped up explaining what I needed to configure with my email provider, same thing with xxx@gmail.com.

I was thinking to myself "If this doesn't even working with gmail the software must be broken", but evidently the issue is that there's no feedback on what's going wrong if you're using a custom domain, and I was unlucky enough to be using two of them.

Peter
